src/main/java/mate/academy/bookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -53,4 +53,13 @@ public ResponseEntity<Object> handleEntityNotFoundException(Exception ex) {
         body.put("errors", ex.getMessage());
         return new ResponseEntity<>(body, HttpStatus.NOT_FOUND);
     }
+
+    @ExceptionHandler(RegistrationException.class)
+    public ResponseEntity<Object> handleRegistrationException(Exception ex) {
+        Map<String, Object> body = new LinkedHashMap<>();
```

Comment:
A lot of code duplication. Extract this responsed building process into separate method or create some class which can be DTO for error response.